filter_test.go

```diff
@@ -168,13 +168,13 @@ func TestAdvancedFilterAddDel(t *testing.T) {
 	}
 
 	u32SelKeys := []TcU32Key{
-		TcU32Key{
+		{
```

Comment:
These changes were made by gofmt, I can split it out into another PR, or a different commit